src/backend/tests/c-lisp/binary-ops.sexp

```diff
@@ -0,0 +1,17 @@
+(c-lisp
+    (define ((print int) (n int)))
+
+    (define ((main int))
+        (declare (five int))
+        (declare (twenty int))
+        (declare (res bool))
+
+        (set five 5)
+        (set twenty 20)
+        (set res (==
```

Comment:
let's create a separate symbol/function sort of thingy for equals. Ideally shouldn't be any sort of casting.

Reply:
Done